Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsAsyncIT.cs

```diff
@@ -88,5 +177,23 @@ public async Task TestReleaseConnectionWhenRollbackFailsAsync()
             // assert
             Assert.AreEqual(0, pool.GetCurrentPoolSize(), "Should not return connection to the pool");
         }
+
+        [Test(Description = "test connection pooling with concurrent connection using async calls")]
+        public void TestConcurrentConnectionPoolingAsync()
+        {
+            // add test case name in connection string to make in unique for each test case
+            // set short expiration timeout to cover the case that connection expired
+            string connStr = ConnectionString + ";application=TestConcurrentConnectionPoolingAsync2;ExpirationTimeout=3";
```

Comment:
does it need to end with "2" - TestConcurrentConnectionPoolingAsync2?

Reply:
this test is actually a performance tests. We have the same tests in ConnectionSinglePoolCacheAsyncIT - I wanted to differentiate application name for these tests.